tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -89,7 +89,7 @@ func NewSetStoreWeightCommand() *cobra.Command {
 // NewStoreLimitCommand returns a limit subcommand of storeCmd.
 func NewStoreLimitCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "limit [<type>]|[<store_id>|<all> <limit> <type>]",
+		Use:   "limit [<type>]|[<store_id>|<all>|<rule> <limit> <type>]",
```

Comment:
Can we make the rule as an option?

Reply:
You mean `store limit all (rule) (limit) (type)`?

Reply:
Someone has also mentioned not to create a new `stores/label/limit` api but to reuse `stores\limit`, I think both ideas can be considered together?